change/@griffel-webpack-extraction-plugin-b04e7a25-7497-4ba1-8e78-2e4163fc33e7.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "feat: export types",
```

```suggestion
  "comment": "feat: export GriffelCSSExtractionPluginOptions type",
```

nit: let's be explicit